### ISSUE TITLE
BUG: Song doesn't play if it was paused for a long time

### ISSUE THREAD

**Issue by SpartakusMd** [Musish/Musish#315]
**Describe the bug**
If a song was paused for more than _let's say 1 hour_ (don't know specific time), then when you press **play**, it doesn't work.

**To Reproduce**
1. Start playing a song
2. Press pause
3. Wait some time
4. Press play

**Expected behavior**
Song starts playing.

**Desktop:**
 - OS: Ubuntu
 - Browser: Chrome
 - Version: 72

**Comment by SpartakusMd** [comment]
The only network call made is `POST https://universal-activity-service.itunes.apple.com/play` but has empty response.
Error found in console 36 times `Uncaught (in promise) DOMException: Failed to execute 'update' on 'MediaKeySession': Rejected with system code (44)`

**Comment by loadedjd** [comment]
Can verify that this becomes an issue after a relatively short period of time being paused (sub 15min.)

**Comment by TheRealContz** [comment]
Same behavior as me. 
On a Mac and with Safari.
Play, pause for a few seconds, the song doesn't restart, you have to skip to the next one in order to start music again.

**Comment by RiFi2k** [comment]
I'm gonna guess this is because the license needed to play the music expires and doesn't get re-inited / recalled. Also alot of the time it returns an empty response with no license.
It's the response from `https://play.itunes.apple.com/WebObjects/MZPlay.woa/wa/acquireWebPlaybackLicense` if I'm correct about how the music is unlocked to be played.

**Comment by RiFi2k** [comment]
Could also be a rate limiting thing on your domain, they don't have very generous rate limits unless you talk to someone over there.

**Comment by BrychanOdlum** [comment]
@RiFi2k Yup, I believe this is the reason too, however I think this is something which MusicKit should be doing itself. I've queried this with them before and will chase up soon.

In the meantime, if we're able to find a workaround that'd be great

**Comment by RiFi2k** [comment]
@BrychanOdlum I have a pretty direct line, well lets say, I happen to have saved the email of the lead dev on musickit.js and I've had a couple conversations with him about things we needed fixed at my day job and our clients that we do work for are high profile enough that everytime I've asked for something so far its been taken care of. If you let me know specifically what you brought to their attention I can see what I can do, no promises of course but might be easier than trying to hack something up if they have the ability to fix it on their side.

**Comment by adrianosbr** [comment]
The same issue affects tracks longer than 15 minutes. It's affecting my own web app (which is focused on classical music, full of very long tracks) and I've just reproduced the behavior in Musish as well.

**Comment by adrianosbr** [comment]
I can confirm that the problem affects Apple's official embedded player (https://tools.applemusic.com/). I issued a bug feedback on Apple's Feedback Assistant. Since it's affecting active players, not only idle players, hopefully it'll receive more attention from them.

**Comment by adrianosbr** [comment]
Apple just released its official Apple Music web player and it suffers with the exact same problem - long tracks always get interrupted at 14'59''.

Additionally: I took a glance at the source code and it seems that Apple is using a v2 of the MusicKitJS SDK:

https://imgur.com/a/648yfWt

**Comment by 1CoolUsername** [comment]
Not sure if this helps any but my current fix is "Dj Bring That Back" which means I click the left arrow twice and hit play. Were going back a couple songs 😎 I believe at times I have been able to just restart the song from the beggining without a reload request. I would have to re-test. Has anyone tried what I have just mentioned?
